src/screens/Introduction/Slide/Slides.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Swiper from "react-native-swiper";
+import HistorySlide from "./HistorySlide";
+import InformationSlide from "./InformationSlide";
+
+import styles from "./styles";
+import ReasonSlide from "./ReasonSlide";
+import TagsSlide from "./TagsSlide";
+
+const Slides = ({ sliderRef, onSliderChanged }) => {
+  return (
+    <Swiper
+      ref={sliderRef}
+      showsButtons={false}
+      loop={false}
+      paginationStyle={styles.box}
+      dotStyle={{ ...styles.dot, ...styles.background }}
+      activeDotStyle={{ ...styles.dot, ...styles.backgroundActive }}
```

Comment:
it's better to place 'standard' backgroundColor inside dot style declaration and apply active background to activeDot:

```
const styles = {
  dot: {
    width: 10,
    height: 10,
    borderRadius: 5,
    backgroundColor: blueLight
  },
  backgroundActive: {
    backgroundColor: blue
  }
};

dotStyle={styles.dot}
activeDotStyle={[styles.dot, styles.backgroundActive]}
```

Reply:
done